geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/z3/XZ3Index.scala

```diff
@@ -9,10 +9,13 @@
 package org.locationtech.geomesa.accumulo.index.z3
 
 import org.locationtech.geomesa.accumulo.AccumuloFeatureIndexType
+import org.locationtech.geomesa.accumulo.index.SplitArrays
 import org.opengis.feature.simple.SimpleFeatureType
 
 case object XZ3Index extends AccumuloFeatureIndexType with XZ3WritableIndex with XZ3QueryableIndex {
 
+  def splitArrays(numSplits: Int): Seq[Array[Byte]] = SplitArrays.getSplitArray(numSplits)
```

Comment:
[picky] we probably don't need all these `def splitArrays` methods in each index class - can just call the `SplitArrays.getSplitArray` directly